### PR TITLE
Re layout the website

### PR DESCRIPTION
**Pull request by machour**
Switching to bootstrap4 and fixing all our layout issues..

### CODE REVIEW

**Comment by samdark** [comment]
What layout issues are fixed? Anything from issues we have open?

**Comment by machour** [comment]
While working on my previous (merged) PR, I noticed a lot of problems with our current CSS:

- a lot of repetitions caused by some bad abstractions
- wrong usage of bootstrap in some views, but things work by accident
- really small base font which make a lot of rules carry a font-size
- usage of a `rem()` utility to convert `px` to `rem` everywhere (which is a non sense to me, you either think and write rems, or pixels)
- lots of wasted place in the mobile UI as the layout was developed desktop-first (generally, as soon as screen is smaller than 1200px, UX is degraded)

So instead of trying to patch these with workarounds like in #430 and #439, I've switched to bootstrap 4 and I'm removing a lot of scss.

The following issues may be solved, or positively impacted by this:
#256, #210, #327, #436, #400, #390, #372, #321, #282, #264, #252, #242, #206, #108

I'll deploy it on my server when I'm done for validation

**Comment by samdark** [comment]
@machour what's the state of this pull request?

**Comment by machour** [comment]
Still needs some work on the docs/guide layout and it should be good to go

File: assets/src/scss/5-pages/_download.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@
   }
 
   pre code {
-    padding: 2rem 4rem;
+  //  padding: 2rem 4rem;
```
Comment (samdark) [inline assets/src/scss/5-pages/_download.scss]: Remove whole block?

**Comment by arogachev** [comment]
What's the status of this PR? @machour @samdark

**Comment by machour** [comment]
@arogachev abandoned. Feel free to close or take over it.

**Pull request closed without merging**